src/lib/holocene/toggle-switch.svelte

```diff
@@ -1,129 +1,34 @@
 <!-- Based loosely on https://www.w3.org/WAI/ARIA/apg/patterns/switch/examples/switch-checkbox/a -->
 <script lang="ts">
-  export let checked = false;
   export let id: string;
-  export let disabled = false;
   export let label: string;
+  export let disabled = false;
+  export let checked = false;
   export let labelPosition: 'left' | 'right' = 'right';
   export let labelHidden = false;
-
-  let focused = false;
-  let hovered = false;
-
-  const handleFocus = () => {
-    focused = true;
-  };
-
-  const handleBlur = () => {
-    focused = false;
-  };
-
-  const handleMouseEnter = () => {
-    hovered = true;
-  };
-
-  const handleMouseLeave = () => {
-    hovered = false;
-  };
 </script>
 
-<label class="switch" class:disabled data-testid={$$props['data-testid']}>
-  {#if labelPosition === 'left'}
-    <span class="label left" class:sr-only={labelHidden}>
-      {label}
-    </span>
-  {/if}
+<label
+  class="group relative flex w-fit min-w-fit cursor-pointer items-center gap-2 rounded px-2 text-primary"
+  class:flex-row-reverse={labelPosition === 'right'}
+  class:pointer-events-none={disabled}
+  data-testid={$$props['data-testid']}
+>
+  <span class="whitespace-nowrap text-sm" class:sr-only={labelHidden}>
+    {label}
+  </span>
   <input
     on:change
+    bind:checked
     {id}
     {disabled}
-    bind:checked
     type="checkbox"
-    class="checkbox"
+    class="peer sr-only"
     role="switch"
-    on:focus={handleFocus}
-    on:blur={handleBlur}
   />
   <span
-    on:mouseenter={handleMouseEnter}
-    on:mouseleave={handleMouseLeave}
-    class="slider"
-    class:disabled
-    class:checked
-    class:focused
-    class:hovered
+    class="surface-subtle not-sr-only flex h-5 w-9 items-center rounded-xl border border-transparent p-px text-primary outline-2 outline-offset-1 outline-brand/50 group-hover:border-brand peer-checked:justify-end peer-checked:bg-brand peer-checked:text-inverse peer-focus:outline peer-disabled:opacity-50"
```

Comment:
What's `p-px` ?

Reply:
Also I noticed there's no hover state when the toggle is "checked", which would be a regression. Were the color changes necessary for dark mode?

Reply:
(Asking cause I followed [the designs](https://www.figma.com/file/LNN3Cu8Akqb240WmJmbtsS/Holocene?type=design&node-id=3177-74002&mode=design&t=TdMPLYu9vYeUA9TV-4) when I implemented this)

Reply:
@rossedfort Heyo, I merged this so I could use something from the plugin.ts that Steve and I were pairing on. I'll go through and double check this matches the new designs and have you review it later today or early tomorrow if that's cool?

Reply:
I think it's OK as is if Ash signed off on it!

Reply:
Just wanted to call attention to the changes